packages/stark-build/config/index-html.transform.js

```diff
@@ -66,5 +66,7 @@ module.exports = (targetOptions, indexHtml) => {
 	    ${indexHtml.slice(i)}`;
 	}
 
-	return replacePlaceholdersByValues(indexHtmlToReturn);
-};
+	return replacePlaceholdersByValues(indexHtmlToReturn, effectiveCommonData);
```

Comment:
Following my comment above, I'd suggest to adapt as following:

```suggestion
	return replacePlaceholdersByValues(indexHtmlToReturn, overriddenCommonData);
```